server/data/data-jmap-draft-cassandra/pom.xml

```diff
@@ -28,10 +28,10 @@
         <relativePath>../../pom.xml</relativePath>
     </parent>
 
-    <artifactId>james-server-data-jmap-cassandra</artifactId>
+    <artifactId>james-server-data-jmap-draft-cassandra</artifactId>
     <packaging>jar</packaging>
 
-    <name>Apache James :: Server :: Data :: JMAP :: Cassandra persistance</name>
+    <name>Apache James :: Server :: Data :: JMAP (draft) :: Cassandra persistence</name>
```

Comment:
Note: Fixed unrelated typo here (persistance -> persistence)

Reply:
Regarding that one, maybe we should not do the same split on persistence APIs.

My thaughts:

 - There are some that both protocols gonna use, and in that case we want the underlying data to keep the same.
 - It's not a bad thing if some APIs are used only by jmap or jmap-draft

I propose to keep it `/server/data/data-jmap-api`...